application.go

```diff
@@ -193,6 +194,7 @@ func (a *Application) SetupLoggerAndConfig() *log.ZapEventLogger {
 	if loadConfigErr != nil {
 		a.logger.Warnf("failed to read config file, creating new one: %v", loadConfigErr)
 	}
+	a.logger.Infof("Anywherelan %s (%s %s-%s)", config.Version, runtime.Version(), runtime.GOOS, runtime.GOARCH)
```

Comment:
I'm little bit scared of using program name without const! Do you think about potential rebranding after reaching popularity?